app/src/main/java/fr/groggy/racecontrol/tv/ui/session/SessionCardPresenter.kt

```diff
@@ -35,8 +41,16 @@ class SessionCardPresenter: Presenter() {
         val view = viewHolder.view as ImageCardView
         val session = item as SessionCard
 
+        val contentSubtype = "session_content_type_" + session.contentSubtype.toLowerCase(Locale.ROOT).replace(" ", "_")
+        val contentSubtypeTranslated: String = try {
+            context.resources.getString(context.resources.getIdentifier(contentSubtype, "string", context.packageName))
+        } catch (e: Exception) {
+            Log.d(TAG, "New translation for $contentSubtype needed")
+            session.contentSubtype
+        }
```

Comment:
HMMM

Let's remove this entire part, it basically introduces a unnecessary complexity in the app that we now need to maintain all the content type by ourselves, while this should be always returned by F1 API in the correct language.